### ISSUE TITLE
Reproduction Results

### ISSUE THREAD

**Issue by itaigat** [jialinwu17/Self_Critical_VQA#7]
Following after the the instructions in the readme file and using the first two pre-trained networks does not reproduce the 49.5% accuracy reported in the paper. 
Also, following the experimental settings in the paper doesn't reproduce the results. 
Tried 20 different seeds and the best score is 49.

Can you please share the exact settings you use to get 49.5? 

Thanks!

**Comment by alice-cool** [comment]
Dear scholar, do you share another download link for train36.h5py. Because I can't download the file using the github link....

**Comment by alice-cool** [comment]
I want to reproduce the result and further research on the visual bias
